packages/gateway-client/src/worker/transformers/local.ts

```diff
@@ -0,0 +1,19 @@
+import { AbstractTransformer } from './transformers';
```

Comment:
This doesn't really solve the problem, since it will only work in a subset of use cases: if we're running on localhost and if the networking service needs to communicate with caddy over localhost:80.

Also, if any of the above are true when we aren't running locally things will also break. What we probably want instead, is to specify a url to caddy in an environment variable that will default to localhost:80.

I can do this in my PR if you want to merge the rest of what is in this one.